game/missiongenerator/aircraft/waypoints/waypointgenerator.py

```diff
@@ -133,9 +134,10 @@ def builder_for_waypoint(self, waypoint: FlightWaypoint) -> PydcsWaypointBuilder
             FlightWaypointType.LOITER: HoldPointBuilder,
             FlightWaypointType.PATROL: RaceTrackEndBuilder,
             FlightWaypointType.PATROL_TRACK: RaceTrackBuilder,
-            FlightWaypointType.PICKUP: CargoStopBuilder,
-            FlightWaypointType.REFUEL: RefuelPointBuilder,
-            FlightWaypointType.STOPOVER: StopoverBuilder,
+            FlightWaypointType.PICKUP_ZONE: LandingZoneBuilder,
+            FlightWaypointType.DROPOFF_ZONE: LandingZoneBuilder,
+            FlightWaypointType.AIR_REFUEL: AirRefuelBuilder,
+            FlightWaypointType.LAND_REFUEL: LandRefuelBuilder,
```

Comment:
Unused after other comments are addressed, I think

Reply:
?

Reply:
nvm, pycharm is failing me